codice/ddf
# DDF-04303 Updated Intrigue configuration names to be consistent

#### What does this PR do?

See the screenshots below.

- Renamed _catalog.ui.config.adoc to catalog.ui.adoc_ to reflect the pid change in #3467
- Split up Intrigue configuration documentation to different adoc files so that they are all listed in the `Search UI Available Configurations` table
- Renamed `Search Email Notifier` to `Catalog UI Search Email Notifier` and fixed some formatting
- Renamed `Workspace Service` to `Catalog UI Search Workspace Service` and added documentation
- Renamed `Workspace Query Monitor` to `Catalog UI Search Workspace Query Monitor` and removed the documentation for the cron configuration property because it was removed in #3812
- Renamed `Workspace Security` to `Catalog UI Search Workspace Security`
- Added documentation for `Catalog UI Search Metacard Type Whitelist`
- Moved `Relevance Score Precision` next to `Show Relevance Scores` in the `Catalog UI Search` configuration
- Improved some formatting and fixed some typos in the Catalog UI Search configurations

#### Who is reviewing it? 

@Kjames5269 @jhunzik @stustison @Lambeaux @bdeining 

#### Select relevant component teams: 

@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.

@ricklarsen - Documentation
@vinamartin 

#### How should this be tested?

- Review the documentation changes ([attached](https://github.com/codice/ddf/files/2928049/documentation.html.zip)).
- Confirm that all of the Intrigue-related config names are prefixed with `Catalog UI Search` in the Admin Console and in the documentation.

#### Any background context you want to provide?

All of the configurations in the `Search UI` app are for Intrigue right now, except `Search UI Redirect`.

#### What are the relevant tickets?

https://github.com/codice/ddf/issues/4303

#### Screenshots

| before | after |
|---|---|
| ![image](https://user-images.githubusercontent.com/8041246/52091353-5c34b100-2571-11e9-856d-200e061e989b.png) | ![image](https://user-images.githubusercontent.com/8041246/52090377-6bfec600-256e-11e9-87d4-32799887d013.png) |
| ![image](https://user-images.githubusercontent.com/8041246/52091196-d31d7a00-2570-11e9-9d78-d17c379e675b.png) | ![image](https://user-images.githubusercontent.com/8041246/52090452-a5cfcc80-256e-11e9-8175-318b5a705e94.png) |
| ![image](https://user-images.githubusercontent.com/8041246/52091233-f5af9300-2570-11e9-961e-3e6934e7596c.png) | ![image](https://user-images.githubusercontent.com/8041246/52090913-fa277c00-256f-11e9-90b6-f2db457997ec.png) |

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
